demo/recommender_system/train_ft.py

```diff
@@ -0,0 +1,137 @@
+import os
+import copy
+
+import paddle.v2 as paddle
+from paddle.v2.reader.creator import cloud_reader
+
+master_ip = os.getenv("MASTER_IP")
+etcd_endpoint = "http://" + master_ip + ":" + "2379"
+
+def get_usr_combined_features():
+    uid = paddle.layer.data(
+        name='user_id',
+        type=paddle.data_type.integer_value(
+            paddle.dataset.movielens.max_user_id() + 1))
+    usr_emb = paddle.layer.embedding(input=uid, size=32)
+    usr_fc = paddle.layer.fc(input=usr_emb, size=32)
+
+    usr_gender_id = paddle.layer.data(
+        name='gender_id', type=paddle.data_type.integer_value(2))
+    usr_gender_emb = paddle.layer.embedding(input=usr_gender_id, size=16)
+    usr_gender_fc = paddle.layer.fc(input=usr_gender_emb, size=16)
+
+    usr_age_id = paddle.layer.data(
+        name='age_id',
+        type=paddle.data_type.integer_value(
+            len(paddle.dataset.movielens.age_table)))
+    usr_age_emb = paddle.layer.embedding(input=usr_age_id, size=16)
+    usr_age_fc = paddle.layer.fc(input=usr_age_emb, size=16)
+
+    usr_job_id = paddle.layer.data(
+        name='job_id',
+        type=paddle.data_type.integer_value(
+            paddle.dataset.movielens.max_job_id() + 1))
+    usr_job_emb = paddle.layer.embedding(input=usr_job_id, size=16)
+    usr_job_fc = paddle.layer.fc(input=usr_job_emb, size=16)
+
+    usr_combined_features = paddle.layer.fc(
+        input=[usr_fc, usr_gender_fc, usr_age_fc, usr_job_fc],
+        size=200,
+        act=paddle.activation.Tanh())
+    return usr_combined_features
+
+
+def get_mov_combined_features():
+    movie_title_dict = paddle.dataset.movielens.get_movie_title_dict()
+    mov_id = paddle.layer.data(
+        name='movie_id',
+        type=paddle.data_type.integer_value(
+            paddle.dataset.movielens.max_movie_id() + 1))
+    mov_emb = paddle.layer.embedding(input=mov_id, size=32)
+    mov_fc = paddle.layer.fc(input=mov_emb, size=32)
+
+    mov_categories = paddle.layer.data(
+        name='category_id',
+        type=paddle.data_type.sparse_binary_vector(
+            len(paddle.dataset.movielens.movie_categories())))
+    mov_categories_hidden = paddle.layer.fc(input=mov_categories, size=32)
+
+    mov_title_id = paddle.layer.data(
+        name='movie_title',
+        type=paddle.data_type.integer_value_sequence(len(movie_title_dict)))
+    mov_title_emb = paddle.layer.embedding(input=mov_title_id, size=32)
+    mov_title_conv = paddle.networks.sequence_conv_pool(
+        input=mov_title_emb, hidden_size=32, context_len=3)
+
+    mov_combined_features = paddle.layer.fc(
+        input=[mov_fc, mov_categories_hidden, mov_title_conv],
+        size=200,
+        act=paddle.activation.Tanh())
+    return mov_combined_features
+
+
+def main():
+    paddle.init(use_gpu=False)
```

Comment:
All the environment variable startswith `PADDLE_INIT` will be converted to the parameterse of `paddle.init`, as https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/__init__.py#L71. And cloud servert will set this environment variable when `-gpu` greater 0, as https://github.com/PaddlePaddle/cloud/blob/develop/paddlecloud/paddlejob/paddle_job.py#L96